scripts/tuple_tree_generator/tuple_tree_generator/templates/tuple_tree_gen.py.tpl

```diff
@@ -65,9 +65,47 @@ class 'enum.name'(EnumBase):
 't' = str
 ## endfor ##
 
+
+class AllMixin:
```

Comment:
These should not be in `tuple_tree_gen.py.tpl`, they should be in a separate file that's passed to `tuple-tree-generate-python.py`